src/libsystemd/sd-id128/id128-util.c

```diff
@@ -41,12 +42,14 @@ bool id128_is_valid(const char *s) {
         return false;
 }
 
-int id128_read_fd(int fd, Id128FormatFlag f, sd_id128_t *ret) {
+int id128_read_at(int dir_fd, const char *path, Id128FormatFlag f, sd_id128_t *ret) {
```

Comment:
The commit message for the commit that introduces this is wrong (says it introduces `id128_write_at()`